gotoalert/strategy.py

```diff
@@ -153,6 +177,13 @@ def get_event_strategy(event):
                 strategy = 'GRB_FERMI_SHORT'
             else:
                 strategy = 'GRB_FERMI'
+    elif event.type == 'NU':
+        if event.notice == 'GOLD':
+            strategy = 'NU_ICECUBE_GOLD'
+        elif event.notice == 'BRONZE': 
+            strategy = 'NU_ICECUBE_BRONZE'
+        elif event.notice == 'CASCADE': 
+            strategy = 'NU_ICECUBE_CASCADE'
```

Comment:
Updated to match event notices in `events.py`.

```suggestion
    elif event.type == 'NU':
        if event.notice == 'ICECUBE_ASTROTRACK_GOLD':
            strategy = 'NU_ICECUBE_GOLD'
        elif event.notice == 'ICECUBE_ASTROTRACK_BRONZE': 
            strategy = 'NU_ICECUBE_BRONZE'
        elif event.notice == 'ICECUBE_CASCADE': 
            strategy = 'NU_ICECUBE_CASCADE'
```